packages/devtools/lib/src/main.dart

```diff
@@ -136,6 +138,19 @@ class PerfToolFramework extends Framework {
     screensReady.future.then(app.devToolsReady);
   }
 
+  void initSseConnection() {
+    try {
+      print('Connecting to SSE endpoint...');
+      final channel = SseClient('/api/sse');
+      channel.onOpen.first.then((e) {
+        channel.stream.listen(print);
+        channel.sink.add('Test message from client');
+      });
+    } catch (e) {
+      print('Failed to connect to SSE API: $e');
```

Comment:
Bonus points for logging this as an error in release (dart2js) builds and only displaying it as a print statement warning is debug (ddk) builds. If debug build I expect the endpoint won't typically be available.

Reply:
Is there any way to tell which version we were built in?

Reply:
yes. replied to the overall thread with an example of detecting whether asserts are enabled.